src/app/scripts/cac/pages/cac-pages-map.js

```diff
@@ -57,8 +61,9 @@ CAC.Pages.Map = (function ($, Handlebars, _, MapControl, Routing, MockDestinatio
 
     return Map;
 
-    function showOptions() {
-        var moreOpt = '.sidebar-options .more-options';
+    function showOptions(event) {
+        var parent = $(event.target).parent().parent();
+        var moreOpt = $('.sidebar-options .more-options', parent);
 
         $(moreOpt).toggleClass('active');
         $(moreOpt).parent().find('a.view-more').text(function() {
```

Comment:
Yes, but how else do I find it relative to the clicked link?  It was toggling both when either tab's expansion was clicked, and I couldn't figure out why `proxy` didn't seem to switch the context.